rascal2/widgets/inputs.py

```diff
@@ -199,6 +199,8 @@ def __init__(self, parent=None):
         self.setStepType(self.StepType.AdaptiveDecimalStepType)
         self.setKeyboardTracking(False)
 
+        self.setDecimals(12)
```

Comment:
It might be better set this to 16 to account for the double precision range  except if there is a reason for the 12

Reply:
I chose the 12 to match up with the code in `textFromValue`: `return f"{round(value, 12):.12g}"`. Happy to change both lines if needed.